File: C++/back2525.cpp
#include <iostream>
#include <string>

using namespace std;

int back2525() {
	int hour = 0;
	int min = 0;
	int plus = 0;

	cin >> hour >> min >> plus;

	int sum = 0;
	sum = min + plus;

	if (sum >= 60) {
		cout << ((hour + (sum / 60)) % 24) << " " << (sum % 60);
	}
	else {
		cout << (hour % 24) << " " << sum;
	}

	return 0;
}

File: Java/back2525.java
import java.util.Scanner;

public class back2525 {

	public static void main(String[] args) {
		
		int hour = 0;
		int min = 0;
		int plus = 0;
		Scanner sc = new Scanner(System.in);
		
		hour = sc.nextInt();
		min = sc.nextInt();
		
		plus = sc.nextInt();
		
		int sum = 0;
		sum = min + plus;
		
		if(sum >= 60) {
			System.out.println(((hour + (sum / 60)) % 24) + " " + (sum % 60));
		}
		else {
			System.out.println((hour % 24) + " " + sum);
		}
		
		
	}

}
